src/silx/gui/plot/CompareImages.py

```diff
@@ -430,53 +445,53 @@ def __separatorMoved(self, pos):
     def clear(self):
         self.setData(None, None)
 
-    def setData(self, image1, image2, updateColormap=True):
+    def setData(self, image1, image2):
```

Comment:
I'd keep compatibility here:
```suggestion
    def setData(self, image1, image2, updateColormap="deprecated"):
```